docs/sdk-librairies/plugins-js-sdk/plugins-apis.md

```diff
@@ -620,55 +622,263 @@ Listener when a participant leaves a room.
 app.onParticipantLeaveRoom = (room, participant) => {};
 ```
 
-## **E-UC Portal** {#portal}
+## **E-UC Mobile** \{#mobile}
 
 ### Methods
 
-#### `changeToolbarDisplay`
+#### `setMobileHeader`
 
-Changes the status/visibility of the toolbar.
+Will change the header inside the app.
 
 ```js
-app.changeToolbarDisplay(displayed: boolean);
+app.setMobileHeader({ title?: string | null, callback?: Function | null });
+```
+
+- `title`: Changes the label displayed in the header. Set this value to `null` to set it back to the default value.
+- `callback`: Handles the back button click. If a callback is registered, the app will forgo its normal behavior and run it. Setting the value to `null` reinstates the original behavior.
+
+#### `setMobileShowBottomNav`
+
+Define whether you want to show or hide the bottom navigation.
+
+```js
+app.setMobileShowBottomNav(show: boolean);
+```
+
+#### `startCall`
+
+Starting a call audio or video call
+
+```js
+app.startCall({ targets , requestedModalities = ['audio'] });
+```
+
+Example:
+
+```js
+app.startCall({ targets: ['8008'], requestedModalities: ['video'] });
+```
+
+#### `playProgressSound`
+
+Play the ringback sound when we make a call (🔁 loop)
+
+```js
+app.playProgressSound();
+```
+
+
+#### `playNewMessageSound`
+
+Play the sound when we receive a text message.
+
+```js
+app.playNewMessageSound();
+```
+
+#### `playHangupSound`
+
+Play the hangup sound
+
+```js
+app.playHangupSound();
+```
+
+#### `stopCurrentSound`
+
+Stop the current sound. Sounds marked "🔁 loop" can be stopped only using this method.
+
+```js
+app.stopCurrentSound();
+```
+
+
+#### `displayBanner`
+
+We can display a mobile banner from the `backgroundScript` file.
+
+```js
+app.displayBanner({ url, height, width, hideCloseButton });
+```
+
+- If `url` is present, the modal will display an iframe with the content of the url.
+- `height` accepts valid CSS values, like `500px` or `80%`.
+- `width` accepts valid CSS values, like `500px` or `80%`, used on Web/Desktop App (min: `300px`).
+- `hideCloseButton` (default to false) determines if the close button should be displayed, or if the user should handle closing of the modal in the html content (through `app.removeModal()` method).
+
+:::note
+On mobile, links with a `target="_blank"` attribute will be opened in the user's default browser.
+:::
+
+#### `closeBanner`
```

Comment:
Duplicate from l.417

Reply:
Yes, it's because we want available methods for both products `Web/Desktop` & `Mobile`  of the documentation.